AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -223,30 +223,18 @@
                                         <span asp-validation-for="SignupModel.AdditionalInfo" class="text-danger"></span>
                                     </div>
                                 </div>
-                                @if (Model.RequiredSkills?.Count > 0)
-                                {
-                                    <div class="form-group">
+                                    <div class="form-group" data-bind="visible: unassociatedSkills.length > 0">

```

Comment:
That's a good point. Fortunately, explicitly comparing a value to `true` or `false` doesn't seem to be a common pattern.